geoips/plugins/modules/readers/amsr2_remss_winds_netcdf.py

```diff
@@ -3,9 +3,20 @@
 
 """Read derived surface winds from REMSS AMSR netcdf data."""
 
+# Python Standard Libraries
+from glob import glob
 import logging
 from os.path import basename
-from glob import glob
+
+# Third-Party Libraries
+import xarray
+
+# GeoIPS-Based imports
```

Comment:
I think this should be `# GeoIPS imports` or `# GeoIPS sub-modules, since the other sections follow that grammar. Also `GeoIPS-Based imports` is improper grammar... Does the tool you're using to standardize this allow you to standardize the comments as well?

Reply:
addressed in [this](https://github.com/NRLMMD-GEOIPS/geoips/commit/b8155a374888d6a5e646107b11a22369aaa88384) commit